src/component/tooltip/tooltipMarkup.ts

```diff
@@ -401,36 +446,46 @@ function wrapBlockHTML(
         + '</div>';
 }
 
-function wrapInlineNameHTML(name: string, leftHasMarker: boolean): string {
+function wrapInlineNameHTML(
+    name: string,
+    leftHasMarker: boolean,
+    style: string
+): string {
     const marginCss = leftHasMarker ? 'margin-left:2px' : '';
-    return `<span style="${TOOLTIP_NAME_TEXT_STYLE_CSS};${marginCss}">`
+    return `<span style="${style};${marginCss}">`
         + encodeHTML(name)
         + '</span>';
 }
 
-function wrapInlineValueHTML(valueList: string[], alignRight: boolean, valueCloseToMarker: boolean): string {
+function wrapInlineValueHTML(
+    valueList: string[],
+    alignRight: boolean,
+    valueCloseToMarker: boolean,
+    style: string
+): string {
     // Do not too close to marker, considering there are multiple values separated by spaces.
     const paddingStr = valueCloseToMarker ? '10px' : '20px';
     const alignCSS = alignRight ? `float:right;margin-left:${paddingStr}` : '';
     return (
-        `<span style="${alignCSS};${TOOLTIP_VALUE_TEXT_STYLE_CSS}">`
+        `<span style="${alignCSS};${style}">`
         // Value has commas inside, so use '  ' as delimiter for multiple values.
         + map(valueList, value => encodeHTML(value)).join('&nbsp;&nbsp;')
         + '</span>'
     );
 }
 
-function wrapInlineNameRichText(ctx: TooltipMarkupBuildContext, name: string): string {
-    return ctx.markupStyleCreator.wrapRichTextStyle(name, TOOLTIP_TEXT_STYLE_RICH);
+function wrapInlineNameRichText(ctx: TooltipMarkupBuildContext, name: string, style: RichTextStyle): string {
+    return ctx.markupStyleCreator.wrapRichTextStyle(name, style as Dictionary<unknown>);
 }
 
 function wrapInlineValueRichText(
     ctx: TooltipMarkupBuildContext,
     valueList: string[],
     alignRight: boolean,
-    valueCloseToMarker: boolean
+    valueCloseToMarker: boolean,
+    style: RichTextStyle
 ): string {
-    const styles: Dictionary<unknown>[] = [TOOLTIP_VALUE_TEXT_STYLE_RICH];
+    const styles: Dictionary<unknown>[] = [style];
```

Comment:
Instead of unknown, use `RichTextStyle`? i.e.:

const styles: Dictionary<RichTextStyle>[] = [style];

Reply:
@chfw  I noticed @100pah is using `Dictionary<unknown>` to represent the style object in this module. I think it's mainly because the style can be both RichTextStyle or normal CSS style for HTML Text.

Reply:
In fact, the accurate type of `ctx.markupStyleCreator.richTextStyles` should be `Dictionary<TextStylePropsPart>`, which is the same as `textStyle.rich`.